src/AutoRest.CSharp/Mgmt/Generation/ResourceOperationWriter.cs

```diff
@@ -94,6 +97,21 @@ private void WriteClientMethods(CodeWriter writer, ResourceOperation resourceOpe
                 writer.Line($"return await ListAvailableLocationsAsync(ResourceType, cancellationToken);");
             }
             writer.Line();
+
+            foreach (var item in context.CodeModel.OperationGroups)
+            {
+                if (item.ParentResourceType(context.Configuration.MgmtConfiguration).Equals(resourceOperation.OperationGroup.ResourceType(context.Configuration.MgmtConfiguration)))
+                {
+                    var container = context.Library.ResourceContainers.First(x => x.ResourceName.Equals(item.Resource(context.Configuration.MgmtConfiguration)));
+                    writer.WriteXmlDocumentationSummary($"Gets a list of {container.ResourceName} in the {resourceOperation.ResourceName}.");
+                    writer.WriteXmlDocumentationReturns($"An object representing collection of {container.ResourceName}s and their operations over a {resourceOperation.ResourceName}.");
+                    using (writer.Scope($"public {container.Type} Get{container.ResourceName}()"))
```

Comment:
need to pluralize the name

GetPolicy => GetPolicies
GetHost => GetHosts
etc

Reply:
Done